src/apprt/gtk/App.zig

```diff
@@ -909,11 +909,8 @@ fn syncConfigChanges(self: *App) !void {
             .{},
         ),
     };
-    self.loadCustomCss() catch |err| switch (err) {
-        error.OutOfMemory => log.warn(
-            "out of memory loading custom CSS, no custom CSS applied",
-            .{},
-        ),
+    self.loadCustomCss() catch |err| {
+        log.warn("Failed to load custom CSS, no custom CSS applied, err={}", .{err});
```

Comment:
We are trying to be more explicit about catching exceptions, so unless something added dozens of errors we should keep the switch. It helps catch regressions if something accidentally starts returning more errors.

Reply:
I implemented your suggestion, however i have some doubts - maybe there is a simpler way to handle the Reader errors. At first I thought handling all errors from an error set would be possible with one case, something like:
```
    self.loadCustomCss() catch |err| switch (err) {
        std.fs.File.Reader.Error => log.warn(
            "failed to load custom CSS, no custom CSS applied - encountered error while reading file: {}",
            .{err},
        ),
    };
```
I tried a few different ways to handle all the errors in one case, however the compiler complained and forced me to handle each error from the set separately, so i just did a one big case:
```
        std.fs.File.Reader.Error.InputOutput,
        std.fs.File.Reader.Error.SystemResources,
        std.fs.File.Reader.Error.IsDir,
        std.fs.File.Reader.Error.OperationAborted,
        std.fs.File.Reader.Error.BrokenPipe,
        std.fs.File.Reader.Error.ConnectionResetByPeer,
        std.fs.File.Reader.Error.ConnectionTimedOut,
        std.fs.File.Reader.Error.NotOpenForReading,
        std.fs.File.Reader.Error.SocketNotConnected,
        std.fs.File.Reader.Error.WouldBlock,
        std.fs.File.Reader.Error.AccessDenied => log.warn(
            "failed to load custom CSS, no custom CSS applied - encountered error while reading file: {}",
            .{err},
        ),
```
If this is not a standard way to handle error sets in switch statements please point me to the correct solution :smile:

Reply:
Yeah, I didn't think that it would get that out of hand. Feel free to revert this part.